src/multibody/joint/joint-planar.hpp

```diff
@@ -459,14 +466,17 @@ namespace se3
 
     ConfigVector_t random_impl() const
     { 
-      ConfigVector_t result(ConfigVector_t::Random());
+      ConfigVector_t result;
+      result.head<2>().setRandom();
+      const Scalar angle = PI * ((Scalar)(-1 + 2 * rand()) / (Scalar) RAND_MAX);
```

Comment:
I realize now I made a mistake here. It should be
```cpp
const Scalar angle = PI * (-1 + 2 * ( (Scalar) rand() / (Scalar) RAND_MAX ) );
```
otherwise `2 * rand()` overflows...